samples/attested_tls/server/enc/Makefile

```diff
@@ -8,6 +8,11 @@ CXXFLAGS=$(shell pkg-config oeenclave-$(CXX_COMPILER) --cflags)
 LDFLAGS=$(shell pkg-config oeenclave-$(CXX_COMPILER) --libs)
 INCDIR=$(shell pkg-config oeenclave-$(C_COMPILER) --variable=includedir)
 
+CLOUD_DEPLOYMENT?=false
+ifeq ($(CLOUD_DEPLOYMENT), true)
+	CXXFLAGS      += -DCLOUD_DEPLOYMENT
```

Comment:
```suggestion
	CXXFLAGS += -DCLOUD_DEPLOYMENT
```

Reply:
Not using compile time flags as per review suggestion.